test/internal/helpers/Player.java

```diff
@@ -846,6 +846,30 @@ public static Cleanups withoutSkill(final int skillId) {
     return new Cleanups(() -> KoLCharacter.removeAvailableSkill(skillId));
   }
 
+  /**
+   * Sets player's substats to given values.
+   *
+   * @param muscle Muscle substats
+   * @param mysticality Mysticality substats
+   * @param moxie Moxie substats
+   * @return Resets substats to zero
+   */
+  public static Cleanups withSubStats(final long muscle, final long mysticality, final long moxie) {
```

Comment:
withStats could be set to just call this function now

Reply:
Done.